hooks/pre-command

```diff
@@ -21,8 +21,10 @@ ecr_exists() {
 upsert_ecr() {
   local repository_name="${1}"
   local max_age_days="${2}"
+  local ecr_tags="${3}"
+
   if ! ecr_exists "${repository_name}"; then
-    aws ecr create-repository --repository-name "${repository_name}"
+    aws ecr create-repository --repository-name "${repository_name}" --tags "${ecr_tags}"
```

Comment:
Should this also update tags one existing repos?

Reply:
I've updated to tag existing repos but not sure how great this is as it is going to do that every time. Is there a better way to do this?

Reply:
Not really, what you have is great